OPEN_PROBLEMS/PRESERVE_USER_PRIVACY.md

```diff
@@ -0,0 +1,121 @@
+# Preseve full users' privacy when providing and fetching Content
+
+## Short Description
+> In one sentence or paragraph.
+
+How to ensure that the user's of the IPFS network can collect and provide information while mainting their full anonymity.
```

Comment:
```suggestion
How to ensure that users of the IPFS network can retrieve and provide information while maintaining full anonymity.
```

Reply:
Thank you @jsoares! You are a saint for reviewing my texts with so much detail, really appreciate it!